params/protocol_params.go

```diff
@@ -254,7 +256,11 @@ func RegionEntropyTarget(expansionNum uint8) *big.Int {
 }
 
 func MinGasLimit(number uint64) uint64 {
-	return 12000000
+	if number < TimeToStartTx {
```

Comment:
wasn this a bug? you had a PR that removed this

Reply:
I don't remember where we landed. Do you want me to keep 12 million as the min gas limit?

Reply:
yeah thats right

Reply:
Fixed